README.md

```diff
@@ -385,15 +408,15 @@ You can centrally create, enforce, and log application and network connectivity
 
 This is designed to support hub and spoke architecture in the azure and further security hardening would be recommend to add appropriate firewall application/network/NAT rules to use this for any production workloads.
 
-All network traffic is directed through the firewall residing in the Management Hub Overlay Network resource group. The firewall is configured as the default route for all other spokes including T0 (Identity and Authorization), T1 (Operations), T2 (Shared Services) Management Spokes and T3 (workload/team environments) workload spoke as follows:
+All network traffic is directed through the firewall residing in the Management Virtual Network Hub Overlay resource group. The firewall is configured as the default route for all other spokes including T0 (Identity and Authorization), T1 (Operations), T2 (Shared Services) Management Spokes and T3 (workload/team environments) workload spoke as follows:
 
 |Name         |Address prefix| Next hop type| Next hop IP address|
 |-------------|--------------|-----------------|-----------------|
 |default_route| 0.0.0.0/0    |Virtual Appliance|10.0.100.4*       |
 
 *-example IP for firewall
 
-The default firewall configured for Management Hub Overlay Network is [Azure Firewall Premium](https://docs.microsoft.com/en-us/azure/firewall/premium-features).
+The default firewall configured for Management Virtual Network Hub Overlay is [Azure Firewall Premium](https://docs.microsoft.com/en-us/azure/firewall/premium-features).
```

Comment:
Would you want to add that Firewall Premium is the only version that meets SCCA compliance as it can break-and-inspect traffic whereas Firewall Standard can't?

Reply:
Added